notebooks/py_scripts/02_Data_Sync.py

```diff
@@ -65,6 +66,15 @@
 import warnings
 
 warnings.filterwarnings("ignore")
+
+
+env_vars = os.environ
+# check that base dir is defined
+assert (
+    "SPYGLASS_BASE_DIR" in env_vars
+), "SPYGLASS_BASE_DIR not set. Please set in your .bashrc or .bash_profile"
+# check that analysis dir is correctly defined relative to base dir
+assert dj.config["stores"]["analysis"]["location"]
```

Comment:
This will also be set on startup [here](https://github.com/LorenFrankLab/spyglass/blob/1062ac8805091d16e89be64d3ff44e40cce45b1b/src/spyglass/settings.py#L175)